app/views/partials/_nav-global.html.erb

```diff
@@ -0,0 +1,6 @@
+<% url_path = request.fullpath.split("/") %>
+<nav data-feature='navToggle' class="global-nav">
+    <a href="#">ground rules</a>
+    <a href="#" class="nav-item<% if url_path[0] == 'machines' %> selected <% end %>" id="machines">Machines</a>
+    <a href="#" class="nav-item<% if url_path[0] == 'roasts' %> selected <% end %>" id="roasts">Roasts</a>
+</nav>
```

Comment:
Same comment as on the _header-global.html.erb file above. This should be moved into the app/views/shared directory